jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessageUtils.kt

```diff
@@ -24,8 +22,9 @@ class MessageIdColumn : ColumnInfo<Message, String>(message("sqs.message.message
 
 class MessageBodyColumn : ColumnInfo<Message, String>(message("sqs.message.message_body")) {
     private val renderer = WrappingCellRenderer(wrapOnSelection = true, toggleableWrap = false)
-    // Truncated the message body to show up to 1KB, as it can be up to 256KB in size. Cannot limit the retrieved message size through API.
-    override fun valueOf(item: Message?): String? = StringUtils.abbreviate(item?.body(), MAX_LENGTH_OF_POLLED_MESSAGES)
+
+    // Truncated the message body to show up to 1024 characters, as it can be up to 256KB in size. Cannot limit the retrieved message size through API.
+    override fun valueOf(item: Message?): String? = item?.body()?.take(MAX_LENGTH_OF_POLLED_MESSAGES)
```

Comment:
We should make Copy copy the raw body instead of truncated

Reply:
Decided to put this in another PR alongside adding a right click copy menu option